vector-im/element-android
# New App Layout FABs

## Type of change


## Content

Adds the new FABs to be used in the New App Layout. These will be hidden behind the New App Layout feature flag

## Motivation and context

Closes https://github.com/vector-im/element-android/issues/6582

## Screenshots / GIFs

| | Feature Flag Off | Feature Flag On |
|-|-|-|
| Light | <img src="https://user-images.githubusercontent.com/20701752/181775694-82a3564c-5123-47ac-8366-ce23d482d1df.png" width="200" /> | <img src="https://user-images.githubusercontent.com/20701752/181774663-e2d7ebf4-a68d-449e-adf9-fcae3625c319.png" width="200" /> |
| Dark | <img src="https://user-images.githubusercontent.com/20701752/181775530-e5fb5d64-b162-4f98-82d0-78f1e8fe759f.png" width="200" /> | <img src="https://user-images.githubusercontent.com/20701752/181774818-e474c12f-eb20-46cb-b520-8fff80e10337.png" width="200" /> |
| Black | <img src="https://user-images.githubusercontent.com/20701752/181775310-84046b24-d517-4711-a5c3-56ad7add64d8.png" width="200" /> | <img src="https://user-images.githubusercontent.com/20701752/181775027-8174b302-170f-47a0-9ed6-6703a3e35bcf.png" width="200" /> |

## Tests



- Enable the New App Layout feature flag
- Sign in and see the FABs

Currently the FABs do not do anything on click. This will be handled in a separate PR

## Tested devices

- OS version(s): Android 11

## Checklist
